plugin_tests/upload_test.py

```diff
@@ -88,6 +88,27 @@ def testUpload(self):
             mimeType='application/zip'
         )
 
+        # Create a new dataset
+        resp = self.request(
+            path='/dataset', method='POST', user=uploaderUser, params={
+                'uploadFolderId': uploadFolder['_id'],
+                'name': 'test_dataset_1',
+                'owner': 'Test Organization',
+                'description': 'A test dataset',
+                'license': 'CC-0',
+                'signature': 'Test Uploader',
+                'anonymous': False,
+                'attribution': 'Test Organization'
+            })
+        self.assertStatusOk(resp)
+        dataset = resp.json
+
+        # Clean folder contents
```

Comment:
Put this into another folder, instead of clearing and re-uploading to the first one.